supertokens_python/recipe/emailpassword/syncio/__init__.py

```diff
@@ -79,12 +79,17 @@ def sign_in(email: str, password: str, user_context: Union[None, Dict[str, Any]]
     return sync(sign_in(email, password, user_context))
 
 
-async def sign_up(email: str, password: str, user_context: Union[None, Dict[str, Any]] = None):
+def sign_up(email: str, password: str, user_context: Union[None, Dict[str, Any]] = None):
     from supertokens_python.recipe.emailpassword.asyncio import sign_up
     return sync(sign_up(email, password, user_context))
 
 
-async def revoke_email_verification_token(user_id: str, user_context: Union[None, Dict[str, Any]] = None):
```

Comment:
This fix should go in another PR to the main x.y branch as it is unrelated to this PR.

Reply:
You need to make another PR fixing this (to the latest X.Y branch).

Reply:
Fixed in https://github.com/supertokens/supertokens-python/pull/168/files